### PR TITLE
This PR updates dependencies and completes the landing page for O…

### PR DESCRIPTION
**Pull request by Princesso**
#### What does this PR do
This PR Updates dependencies and completes the landing page for OSCAFest 2023.

#### Description of Task to be completed
- Update dependencies 
- Complete the landing page for OSCAFest 2023.
- Move 2022 event to new folder

#### How should this be manually tested
Look at this [design](https://www.figma.com/file/uGbiNNx3nZqONJ7qBKORkG/website-%26-stage?node-id=142%3A3&t=lZyl8dSZ4wEvGlr7-0) and compare both

#### What are the relevant issues this PR belongs to
NA

### CODE REVIEW

**Review by BolajiAyodeji — COMMENTED** [review]
Thanks for this, PR chief. Here are some comments from my review:

- There's a slight difference with the date/location section; it doesn't seem centered and aligned accurately, as seen below.

| Design | Code (this PR) |
| - | - |
| Screenshot 2023-03-14 at 09 27 51 | Screenshot 2023-03-14 at 09 27 18 |

- The UI is broken for previous years—mostly the desktop screen, probably due to the change in Tailwind configuration and additional screens. You can compare them yourself (2022: [current](https://festival.oscafrica.org) | [this PR](https://deploy-preview-166--osca-festival.netlify.app/2022) and 2020: [current](https://festival.oscafrica.org/2020/) | [this PR](https://deploy-preview-166--osca-festival.netlify.app/2020)).
- The font `Clash Display Variable` used in the design isn't implemented yet (you can get the files [here](https://fontsfree.net/clash-display-variable-bold-font-download.html)). Also, the fonts in the current implementation are not the same font weight and size as seen in the design.
- The current `VIEW PREVIOUS EVENTS` should be `VIEW PAST EVENTS` according to the design (not sure if this was intentional).
- I believe this button should link back to the `previous-events` page when you get the design for that, but if you consider changing the text back to `VIEW PAST EVENTS`, then maybe the slug of this page should be `past-events`.
- Suggestion: the `::-webkit-scrollbar-thumb`, `::selection`, and `::-moz-selection` selectors use the color scheme from last year, which affects every year, including this PR. You can consider using a neutral color that would match all, irrespective of their unique themes.

Aside from these minor issues, I think everything else looks good! If the comments are reverted, I will proceed to test locally.

**Comment by BolajiAyodeji** [comment]
I'd also allow @perriefidelis sign off on the design itself.

**Comment by Princesso** [comment]
@BolajiAyodeji

Thanks for reviewing the PR. 

Just to let you know, all the issues you noted have been fixed. Can you have another look? In case I need to include or remove anything.

**Comment by BolajiAyodeji** [comment]
> @BolajiAyodeji
> 
> Thanks for reviewing the PR.
> 
> Just to let you know, all the issues you noted have been fixed. Can you have another look? In case I need to include or remove anything.

Please can you check; the build failed?

File: gatsby-config.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ module.exports = {
   siteMetadata: {
     title: "Open Source Festival",
     description: "A high profile event with series of talks, workshops, and awareness of open-sourced developer tools.",
-    siteUrl: "https://festival.oscafrica.org",
+    // siteUrl: "https://festival.oscafrica.org",
+    siteUrl: "https://deploy-preview-166--osca-festival.netlify.app/",
```
Comment (BolajiAyodeji) [inline gatsby-config.js]: Should this be here?

**Review by BolajiAyodeji — COMMENTED** [review]
The icons in the buttons doesn't seem to be spaced well as compared to the design.

Screenshot 2023-03-15 at 22 39 08

Aside from the comments above, this looks good to me!

**Review by BolajiAyodeji — APPROVED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (BolajiAyodeji) [reply gatsby-config.js]: @Princesso you forgot to remove this?

**Review by alabobriggs — COMMENTED** [review]
Looks good.

@Princesso for the next PRs, try to keep changes small < 2000 line changes. 

So its easier to review what exactly was changed and a more streamlined UI testing.

**Comment by BolajiAyodeji** [comment]
@all-contributors please add @Princesso for code

**Pull request merged**